app/templates/views/security.html

```diff
@@ -1,89 +1,93 @@
+    <p class="govuk-body">If you’re sending a message with sensitive information, you can choose to hide those details on the Emergency Alerts dashboard once the message has been sent. This means that only the message recipient will be able to see that information.</p>
 
-  <h2 class="heading-medium" id="user-permissions-signing-in">User permissions and signing in</h2>
-  <p class="govuk-body">You can set different user permissions in Notify. This lets you control who in your team has access to certain parts of the service.</p>
-  <h3 class="heading-small">Two-factor authentication</h3>
-  <p class="govuk-body">To sign in to Notify, you’ll need to enter:</p>
-  <ul class="govuk-list govuk-list--bullet">
-    <li>your email address and password</li>
-    <li>a text message code that Notify sends to your phone</li>
-  </ul>
-  <p class="govuk-body">If signing in with a text message is a problem for your team, <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.support') }}">contact us</a> to find out about using an email link instead.</p>
+    <h2 class="heading-medium" id="user-permissions-signing-in">User permissions and signing in</h2>
+    <p class="govuk-body">You can set different user permissions in Emergency Alerts. This lets you control who in your team has access to certain parts of the service.</p>
+    <h3 class="heading-small">Two-factor authentication</h3>
+    <p class="govuk-body">To sign in to Emergency Alerts, you’ll need to enter:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>your email address and password</li>
+      <li>a text message code that Emergency Alerts sends to your phone</li>
+    </ul>
+    <p class="govuk-body">If signing in with a text message is a problem for your team, <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.support') }}">contact us</a> to find out about using an email link instead.</p>
 
-  <h2 class="heading-medium" id="information-risk-management">Information risk management</h2>
-  <p class="govuk-body">Our approach to information risk management follows NCSC guidance. It assesses:</p>
-  <ul class="govuk-list govuk-list--bullet">
-    <li>how Notify is built</li>
-    <li>the infrastructure Notify is built upon</li>
-    <li>support for the Notify service</li>
-  </ul>
-  <p class="govuk-body">This approach also applies to the service providers Notify uses to send messages.</p>
+    <h2 class="heading-medium" id="information-risk-management">Information risk management</h2>
+    <p class="govuk-body">Our approach to information risk management follows NCSC guidance. It assesses:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>how Emergency Alerts is built</li>
+      <li>the infrastructure Emergency Alerts is built upon</li>
+      <li>support for the Emergency Alerts service</li>
+    </ul>
+    <p class="govuk-body">This approach also applies to the service providers Emergency Alerts uses to send messages.</p>
```

Comment:
We don't use service providers directly - we could change this to something like
"This approach also applies to the Notify service that the Emergency Alerts service uses to send emails and SMS messages for account set-up and two-factor authentication."